framework/src/modules/network/index.js

```diff
@@ -49,12 +49,26 @@ module.exports = class NetworkModule extends BaseModule {
 
 	get actions() {
 		return {
-			request: async action => this.network.actions.request(action),
-			emit: action => this.network.actions.emit(action),
-			getNetworkStatus: () => this.network.actions.getNetworkStatus(),
-			getPeers: action => this.network.actions.getPeers(action),
-			getPeersCountByFilter: action =>
-				this.network.actions.getPeersCountByFilter(action),
+			request: {
+				handler: async action => this.network.actions.request(action),
+				isPublic: false,
+			},
+			emit: {
+				handler: action => this.network.actions.emit(action),
+				isPublic: false,
+			},
+			getNetworkStatus: {
+				handler: () => this.network.actions.getNetworkStatus(),
+				isPublic: true,
```

Comment:
this is also should be false

Reply:
No, it's currently needed for discovering peers as this also holds a list of the connected peers.
Without, network tests fail, but @jondubois is changing this. When he's doing his PR, he should make this false again.